databuilder/models/table_metadata.py

```diff
@@ -177,13 +244,14 @@ def __init__(self,
         :param columns:
         :param is_view: Indicate whether the table is a view or not
         :param tags:
+        :param description_source: Optional. Where the description is coming from. Used as a unique id.
```

Comment:
How about `s/Used as a unique id/Used to compose unique id`?

Reply:
changed